tests/composer.json.travis

```diff
@@ -0,0 +1,170 @@
+{
+  "name": "PrestaShop/PrestaShop",
+  "description": "PrestaShop offers a free, fully scalable, Open Source e-commerce solution.",
+  "type": "project",
+  "repositories": [
+    {"type": "vcs", "url": "https://github.com/prestashop/blockcart"},
```

Comment:
Yes, I just saw that. I've removed some lines a long time ago and didn't remember there was still much :/ 
Sorry